packages/metrics/src/node/metrics-contribution.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as prom from 'prom-client';
+import { injectable } from 'inversify';
+
+export const MetricsContribution = Symbol("MetricsContribution");
+export interface MetricsContribution {
+    startCollecting(): void;
+    getMetrics(): string;
+}
+
+@injectable()
+export class PrometheusContribution implements MetricsContribution {
```

Comment:
This should be NodeMetricsContribution since Prometheus is the export format... not what is exported.